.ci_support/linux_64_.yaml

```diff
@@ -1,7 +1,3 @@
-c_stdlib:
```

Comment:
Is this expected?

Reply:
This is from re-rending the feedstock, I think it is related to the stdlib migration https://conda-forge.org/news/2024/03/24/stdlib-migration/